run-tests.sh

```diff
@@ -106,9 +107,15 @@ test_service() {
 }
 
 test_file_system() {
+  docker run -d -p 9090:9090 -p 9191:9191 --name $S3MOCK_DOCKER_NAME -t adobe/s3mock
+    ./gradlew clean check fileSystemITest -Dspring.datasource.url=$JDBC_URL -Dfile.processing.decompress=true -Dirida.it.rootdirectory=$TMP_DIRECTORY -Dspring.datasource.dbcp2.max-wait=$DB_MAX_WAIT_MILLIS $@
```

Comment:
Please remove this duplicate execution of the tests. line 111 and 112

Reply:
Removed in 52e42e91096b76089f29d43565068647ca7dc85c. Good catch :)